lib/sisimai/bite/email/exim.rb

```diff
@@ -295,9 +326,16 @@ def scan(mhead, mbody)
                   else
                     # Error message when email address above does not include '@'
                     # and domain part.
-                    next unless e =~ /\A[ ]{4}/
-                    v['alterrors'] ||= ''
-                    v['alterrors']  += e + ' '
+                    if e =~ %r<\A[ ]+pipe[ ]to[ ][|]/.+>
```

Comment:
__[RuboCop]__

Style/IfInsideElse: Convert `if` nested inside `else` to `elsif`.


<!-- Comment by [SideCI](https://sideci.com) -->